draft-parecki-oauth-first-party-native-apps.md

```diff
@@ -421,10 +436,18 @@ Possible, but of scope of this document.
 
 ## Multiple Applications {#multiple-applications}
 
-* Increased phishing risk
-* User confusion
-* Increased attack surface
-* Higher chance of incorrect implementations
+When there there is more than one 1st-party native applications supported by the AS, then it is important to consider a number of additional risks. These risks fall into two main categories: Experience Risk and Technical Risk which are described below.
+
+### Experience Risk
+Any time a user is asked to provide the authentication credentials in user experiences that differ, it has the effect of increasing the likelihood that the user will fall prey to a phishing attack because they are used to entering credentials in different looking experiences. When multiple native applications are support, the implementation MUST ensure the native experience is identical across all the 1st party native applications.
+
+Another experience risk is user confusion caused by different looking experiences and behaviors. This can increase the likelihood the user will fail the authentication experience for the 1st party native application.
```

Comment:
```suggestion
Another experience risk is user confusion caused by different looking experiences and behaviors. This can increase the likelihood the user will not complete the authentication experience for the 1st party native application.
```